.github/workflows/openrpc-updater.yml

```diff
@@ -0,0 +1,138 @@
+
+name: OpenRPC JSON Updater
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'docs/openrpc.json'
+
+jobs:
+  clone-and-build-execution-apis:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout execution-apis repo
+        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+        with:
+          ref: main
+          repository: 'ethereum/execution-apis'
+          path: 'execution-apis'
+
+      - name: Use Node.js TLS 20
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version: 20
+
+      - name: Install dependencies
+        run: npm install
+        working-directory: ./execution-apis
+
+      - name: Build project
+        run: npm run build
+        working-directory: ./execution-apis
+
+      - name: Upload openrpc.json as an artifact
+        uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4.3.3
+        with:
+          name: openrpc
+          path: ./execution-apis/refs-openrpc.json
```

Comment:
We might include a brief description somewhere on the difference between `refs-openrpc.json` and `openrpc.json`. The execution-apis repo doesn't clarify what `refs-openrpc.json` is. Perhaps in the `README.md`?